client/src/js/app.js

```diff
@@ -74,3 +76,5 @@ class App {
 }
 const app = new App(sources, settings)
 app.init()
+const newsHeadings = new Headlines(settings)
```

Comment:
I would suggest to move somehow into you App since semantically it is one application and it will be better to take in one place